apps/api/tasks.py

```diff
@@ -49,14 +47,14 @@ def setup_connect_channels_for_bots(self, connect_id: UUID, experiment_data_map:
     # https://github.com/dimagi/open-chat-studio/issues/1046
     channels = ExperimentChannel.objects.filter(
         platform=ChannelPlatform.COMMCARE_CONNECT,
-        experiment_id__in=[participant_data.object_id for participant_data in participant_data],
+        experiment_id__in=[participant_data.experiment_id for participant_data in participant_data],
     )
 
     channels = {ch.experiment_id: ch for ch in channels}
```

Comment:
Not directly related to this PR but since we're doing this already it feels like we could get rid of the `experiments_using_connect` subquery above and just use a static list of experiment IDs (by moving this above the other query).

Reply:
Sure. I'll add it to this [PR changes](https://github.com/dimagi/open-chat-studio/pull/1071)

Reply:
Looking at you comment again, I am a bit confused by what you're suggesting. Are you suggesting we use the `experiment_ids` list to get these channels?